src/interface.rs

```diff
@@ -706,7 +761,7 @@ impl TryFrom<&Route> for IpRoute {
             };
             IpInet::new(default_ip, 0)?
         } else {
-            return Err(anyhow::anyhow!("Error occurred when parsing route"));
+            return Err(anyhow::anyhow!("Error occurred when parsing {route}"));
```

Comment:
Personally I would say this whole formatter is overkill for something that is very unlikely to occur. Maybe just something like
```
Route contains neither destination nor next_hop
```
with my other suggestion the result would e.g. be
```
Failed parsing route for eth0: Route contains neither destination nor next_hop
```
which IMO is good enough.

Reply:
just added `ipv4` and `ipv6` to indicate which one...